candi/image_versions.yml

```diff
@@ -4,7 +4,7 @@
 REGISTRY_PATH: "registry.deckhouse.io/base_images/"
 
 BASE_ALPINE: "alpine:3.20.3@sha256:41628df7c9b935d248f64542634e7a843f9bc7f2252d7f878e77f7b79a947466"
-BASE_GOLANG_23_ALPINE: "golang:1.23.6-alpine3.20@sha256:3058c63e0e2532881949c4186414baa24a0f9a8f9349b1853daa49be816f42e9"
+BASE_GOLANG_23_ALPINE: "golang:1.23.8-alpine3.20@sha256:7155e5b534ac02fb85191c51d64f727060a11ae987899f2b60c109c93f2c4777"
```

Comment:
use new base images, and i think we can use 1.24 go

Reply:
We have to build deckhouse-controller. This variable will not affect it.
https://github.com/deckhouse/deckhouse/blob/57c4ce2d5a2d2af6a547f526108995e7430a69af/.werf/defines/base.tmpl#L137